docs/articles_en/assets/snippets/ov_stateful_model_intro.py

```diff
@@ -0,0 +1,208 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import logging as log
+import numpy as np
+
+import openvino  as ov
+from openvino.runtime import opset13 as ops
+from openvino.runtime.op.util import VariableInfo, Variable
+from openvino.runtime.passes import LowLatency2, MakeStateful, Manager
+from openvino.runtime.utils import replace_node
+
+
+def state_network_example():
+    #! [ov:state_network]
```

Comment:
it's better to rename "[ov:state_network]" to "[ov:stateful_model]"

Reply:
done for c++ as well